src/genericComponents/Input.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { FormControl, InputGroup } from 'react-bootstrap';
+import styled from 'styled-components';
+
+const InputField = (props: any) => {
+  const { placeholder } = props;
```

Comment:
We're going to need a lot more props than this. You're also not using placeholder below.